tests/sentry/plugins/test_integration_repository.py

```diff
@@ -81,3 +83,13 @@ def test_create_repository__delete_webhook(self, get_jwt, mock_on_delete, mock_r
 
         with pytest.raises(RepoExistsError):
             self.provider.create_repository(self.config, self.organization)
+
+    @patch("sentry.integrations.github.client.get_jwt", return_value=b"jwt_token_1")
+    def test_create_repository__activates_existing_repo(self, get_jwt):
```

Comment:
Once you filter for HIDDEN, add another test case to make sure we only change hidden repos

Reply:
we raise the `RepoExists` error if the repo exists but is not hidden. i could swallow the exception instead?